worker/introspection/leases.go

```diff
@@ -187,3 +199,138 @@ func (h leaseHandler) format(leases *leases) map[string]interface{} {
 	}
 	return result
 }
+
+// ServeHTTP is part of the http.Handler interface.
+func (h *leaseHandler) revoke(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		fmt.Fprintln(w, "revoking a lease requires a POST request")
```

Comment:
Can we add the method received to this log?

Reply:
yup